OpenEdXMobile/res/values-es/strings.xml

```diff
@@ -467,8 +467,6 @@ Si no lo recibe en algunos minutos, asegúrese de revisar su carpeta de spam.</s
   <string name="launch_register">Crea tu Cuenta</string>
   <!--Button leading user to log in screen-->
   <string name="launch_login">Iniciar sesión</string>
-  <!--Introduction text on the launch screen-->
-  <string name="launch_text">Cursos gratuitos de las mejores universidades del mundo en tu bolsillo.</string>
```

Comment:
> is it necessary to remove the translation?

Yes, otherwise in the Spanish language the text won't be the updated one.

> I think need to update the translation as well, because not mentioned in the story.

For that, we'll have to rely on our translators to do it once this PR merges and the new string is available for them to translate.